evaluate-artifact-with-expression-with-output broken or flaky

### Affected version

HEAD

### Bug description

flaky or broken IT

```
*** end build.log for: evaluate-artifact-with-expression-with-output/pom.xml ***

[ERROR] -------------------------------------------------
[ERROR] 
[INFO] -------------------------------------------------
[INFO] Build Summary:
[INFO]   Passed: 32, Failed: 1, Errors: 0, Skipped: 2
[INFO] -------------------------------------------------
[ERROR] The following builds failed:
[ERROR] *  evaluate-artifact-with-expression-with-output/pom.xml
[INFO] -------------------------------------------------
[WARNING] The following builds were skipped:
[WARNING] *  evaluate-settings-servers/pom.xml
[WARNING] *  active-profiles_multimodule/pom.xml
```

--- a/src/it/projects/effective-pom-artifact/pom.xml
+++ b/src/it/projects/effective-pom-artifact/pom.xml
@@ -27,4 +27,16 @@ under the License.
   <version>1.0-SNAPSHOT</version>
   <description>https://issues.apache.org/jira/browse/MPH-106</description>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-help-plugin</artifactId>
+        <version>@project.version@</version>
+        <configuration>
+          <artifact>org.apache.maven.plugins:maven-help-plugin:@project.version@</artifact>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
 </project>

--- a/src/it/projects/effective-pom-artifact/test.properties
+++ b/src/it/projects/effective-pom-artifact/test.properties
@@ -16,4 +16,3 @@
 # under the License.
 
 output = result.txt
-artifact = org.apache.maven.plugins:maven-help-plugin

--- a/src/it/projects/effective-pom-multimodule-artifact/pom.xml
+++ b/src/it/projects/effective-pom-multimodule-artifact/pom.xml
@@ -36,6 +36,14 @@ under the License.
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-help-plugin</artifactId>
+        <version>@project.version@</version>
+        <configuration>
+          <artifact>org.apache.maven.plugins:maven-help-plugin:@project.version@</artifact>
+        </configuration>
+      </plugin>
     </plugins>
   </build>
 </project>

--- a/src/it/projects/effective-pom-multimodule-artifact/test.properties
+++ b/src/it/projects/effective-pom-multimodule-artifact/test.properties
@@ -16,4 +16,3 @@
 # under the License.
 
 output = result.txt
-artifact = org.apache.maven.plugins:maven-help-plugin

--- a/src/it/projects/evaluate-artifact-with-expression-with-output/pom.xml
+++ b/src/it/projects/evaluate-artifact-with-expression-with-output/pom.xml
@@ -41,6 +41,9 @@ under the License.
               <goal>evaluate</goal>
             </goals>
             <phase>package</phase>
+            <configuration>
+              <artifact>org.apache.maven.plugins:maven-help-plugin:@project.version@</artifact>
+            </configuration>
           </execution>
         </executions>
       </plugin>

--- a/src/it/projects/evaluate-artifact-with-expression-with-output/test.properties
+++ b/src/it/projects/evaluate-artifact-with-expression-with-output/test.properties
@@ -17,4 +17,3 @@
 
 expression = project.name
 output = result.txt
-artifact = org.apache.maven.plugins:maven-help-plugin

